po/es.po

```diff
@@ -62,7 +64,7 @@ msgstr "Gobernador"
 #: hardinfo/cpu_util.c:190 hardinfo/cpu_util.c:196 modules/devices/gpu.c:153
 #: modules/devices/pci.c:134 modules/devices/usb.c:136
 msgid "Driver"
-msgstr "Manejador"
+msgstr "Controlador"
```

Comment:
this are a win32 only concept! really are controlling or just ask to OS to control?

Reply:
At least in Spain, "Controlador" is the translation for "Driver", "Manejador" is seldom used here. Try "Device driver" in Google Translate, for example.

Reply:
Google translation are for windoser user! there's not such in hurd or bsd and hardinfo compiles for those systems!

Reply:
I'm really having a hard time trying to understand some of your comments.

What's that "Google translation are for windowser user"? A driver is a piece of software that sits between the OS and the hardware. It doesn't matter if you use Linux, Windows or Amoeba. I've used MSDOS, Windows, Linux, QNX, and probably some more in the university and driver was a concept used in all of them. If you don't like Google Translator or DeepL or whatever, get a computing text book and check it for yourself.

We can agree or disagree about the translation but that doesn't change what a driver is. "Manejador" is used more frequently in South America and for me it's ok if we use it, no problem.